### ISSUE TITLE
MD029 triggered for ordered lists.

### ISSUE THREAD

**Issue by dalee-bis** [DavidAnson/vscode-markdownlint#20]
The MD029 rule is being triggered by ordered lists despite this being described as valid by https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md029

Example code:
```
1. Item 1
2. Item 2
3. Item 3
```

VSCode Version: 1.18.1
Markdownlint Version: 0.11.1

**Comment by DavidAnson** [comment]
The default mode is “one”, please see my reply here for more context: https://github.com/DavidAnson/markdownlint/issues/45

**Comment by DavidAnson** [comment]
I’m thinking I may update the error message to show the current setting to make the default more obvious.

**Comment by DavidAnson** [comment]
The following update should help clarify the current mode:
https://github.com/DavidAnson/markdownlint/commit/f910fa0cb1b13f45aad75d7590f3d8e7c0fd8778

**Comment by dalee-bis** [comment]
Apologies, I hadn't realised there were parameters for the rules.

For anyone else who comes across this, I fixed it by adding the following into my workspace settings:
```
    "markdownlint.config": {
        "MD029": {
            "style":"ordered"
        }
    }
```

**Comment by DavidAnson** [comment]
No worries, I hope the updated message helps a little!
